.env.example

```diff
@@ -30,7 +30,7 @@ RELAYER_STORAGE_PATH=storage/leveldb
 RELAYER_QUERIES_TASK_QUEUE_CAPACITY=10000
 RELAYER_CHECK_SUBMITTED_TX_STATUS_DELAY=10s
 RELAYER_INITIAL_TX_SEARCH_OFFSET=0
-RELAYER_PROMETHEUS_PORT=9999
+RELAYER_WEBSERVER_PORT=9999
```

Comment:
I believe it should be possible to define not just a port, but a whole URL to bind to, e.g. `127.0.0.1:9999`